### PR TITLE
Using go 1.5.1

### PR DESCRIPTION
**Pull request by ewoutp**
This PR changes the go version use to compile `swarm`.
It switches from 1.4.2 to 1.5.1.

### CODE REVIEW

**Comment by ewoutp** [comment]
@giantswarm/team-features RFR

File: Makefile [hunk 1]
```diff
@@ -92,11 +93,20 @@ $(BIN): VERSION $(SOURCE)
...
+	docker run \
+	    --rm \
+	    -v $(shell pwd):/usr/code \
+	    -e GOPATH=/usr/code/.gobuild \
+	    -e GOOS=$(GOOS) \
+	    -e GOARCH=$(GOARCH) \
+	    -e CGO_ENABLED=0 \
+	    -w /usr/code \
+	    golang:$(GOVERSION) \
+		go test ./...
```
Comment (zyndiecate) [inline Makefile]: Is this indention on purpose?

**Comment by zyndiecate** [comment]
LGTM

File: Makefile (hunk 1 above)
Comment (ewoutp) [inline Makefile]: Crap, my editor again

**Pull request merged**